AssemblerLib/LocalToGlobalIndexMap.h

```diff
@@ -68,6 +68,12 @@ class LocalToGlobalIndexMap
     /// Returns total number of degrees of freedom.
     std::size_t dofSize() const;
 
+    /// Returns total number of local degrees of freedom for DDC.
+    std::size_t dofSizeLocal() const
+    {
+        return _mesh_component_map.getNLocalUnknowns();
+    }
+
     /// Returns total number of global degrees of freedom for DDC.
     std::size_t dofSizeGlobal() const
```

Comment:
A question for understanding: what is the difference between `dofSize` and `dofSizeGlobal` in case of domain decomposition? One is the mesh component's dictionary size, the other one is like number of global mesh nodes used in the mesh component map...